src/bun.js/bindings/ZigGlobalObject.cpp

```diff
@@ -1496,7 +1496,7 @@ JSC_DEFINE_HOST_FUNCTION(functionPerformanceNow, (JSGlobalObject * globalObject,
     auto* global = reinterpret_cast<GlobalObject*>(globalObject);
     // nanoseconds to seconds
     uint64_t time = Bun__readOriginTimer(global->bunVM());
-    double result = time / 1000000000.0;
+    double result = time / 1000000000000000.0;
     return JSValue::encode(jsNumber(time));
```

Comment:
You are still returning the time variable. Please calculate double result = time / 1000000.0; and change jsNumber(time) to jsNumber(result)

Reply:
I overlooked that, must the added 0's are incorrect and should be time / 1_000_000.0 I suppose